samples-js/functions/httpSyncStart.js

```diff
@@ -0,0 +1,39 @@
+const { app } = require("@azure/functions");
+const df = require("durable-functions");
+
+const timeout = "timeout";
+const retryInterval = "retryInterval";
+
+const clientInput = df.input.durableClient();
+
+app.http("httpSyncStart", {
+    methods: ["POST"],
+    route: "orchestrators/wait/{orchestratorName}",
+    authLevel: "anonymous",
+    extraInputs: [clientInput],
+    handler: async function (request, context) {
+        const client = df.getClient(context, clientInput);
+        const body = await request.json();
+        const instanceId = await client.startNew(request.params.orchestratorName, undefined, body);
+
+        context.log(`Started orchestration with ID = '${instanceId}'.`);
+
+        const timeoutInMilliseconds = getTimeInSeconds(request, timeout) || 30000;
+        const retryIntervalInMilliseconds = getTimeInSeconds(request, retryInterval) || 1000;
+
+        const response = client.waitForCompletionOrCreateCheckStatusResponse(
+            request,
+            instanceId,
+            timeoutInMilliseconds,
+            retryIntervalInMilliseconds
+        );
+        return response;
+    },
+});
+
+function getTimeInSeconds(req, queryParameterName) {
+    const queryValue = req.query.get(queryParameterName);
+    return queryValue
+        ? queryValue * 1000 // expected to be in seconds
+        : undefined;
+}
```

Comment:
Is it "expected" to be in seconds, or is it just "in seconds"?

Reply:
I think the "expected" here was referring to `queryValue` being in seconds, and it is "expected" because the user can pass in anything they want. That being said, I renamed the function name and changed the comments so hopefully it's more readable now